pkg/scheduler/scheduler_test.go

```diff
@@ -19,13 +19,16 @@ package scheduler
 import (
 	"context"
 	"fmt"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
nit: move it down with other k8s.io imports.

Reply:
fixed.